src/main/kotlin/com/autonomousapps/tasks/FindDeclarationsTask.kt

```diff
@@ -85,40 +82,20 @@ abstract class FindDeclarationsTask : DefaultTask() {
     ): Sequence<Configuration> {
       val seq = configurations.asSequence()
         .filter { it.isForRegularDependency() || it.isForAnnotationProcessor() }
-        .filterNot {
-          // this is not ideal, but it will solve some problems till we can support androidTest analysis.
-          // will match, e.g., "androidTestImplementation", "debugAndroidTestImplementation", and "kaptAndroidTest".
-          it.name.contains("androidTest", true)
-        }
```

Comment:
This is no longer needed, because it is covered by the more general "supported source sets" implementation.